### DISCUSSION TITLE
add_requires报错（fatal: not a git repository） 该怎么解决？

### DISCUSSION THREAD

**Discussion by cpp-next** [xmake-io/xmake#2308] [Q&A] [answered]
xmake.lua内容：

add_requires("vcpkg::zlib", {alias = "zlib"})

target("xmake-stu")
    set_kind("binary")
    add_files("src/*.cpp")
    set_languages("c99", "cxxlatest")

报错截图：
![image](https://user-images.githubusercontent.com/81665528/165085116-0b06d585-9803-41e5-8b30-abb25cfe472d.png)

**Comment by waruqi** [comment, marked as answer]
手动删下 `%localappdata%/.xmake/repositories` 目录试试

**Reply by cpp-next** [reply]
谢谢，已解决。

**Reply by jpc0** [reply]
For those english people that came here, try to manually delete `%localappdata%/.xmake/repositories`
